trench/models.py

```diff
@@ -70,6 +71,8 @@ class MFAMethod(Model):
     )
     name = CharField(_("name"), max_length=255)
     secret = CharField(_("secret"), max_length=255)
+    counter = PositiveIntegerField(_("counter"), default=0)
```

Comment:
Perhaps a unix epoch would be a better counter to use. It auto increments (just set the counter to the current time when generating a code) and allows us to apply a validity window (count <= now < =counter + validity window) without needing to store an additional code creation timestamp (not that the data cost would be very significant).